src/notes.gleam

```diff
@@ -50,6 +52,53 @@ pub fn main() {
         }
       },
     )
+    |> router.delete(
+      "/delete/:id",
+      {
+        fn(req: Request(BitString, assigns, session)) {
+          case request.get_param(req, "id") {
+            Some(id) -> {
+              case int.parse(id) {
+                Ok(id) -> {
```

Comment:
Is there no way to make this less verbose? Could the path params be validated in a middleware?

Reply:
I updated the types on the pgo wrapper so we can flatten the results but the params stuff needs some investigation. 

Ideally we could eliminate the `_ -> Error(Nil)` if param not found because it's impossible for it to hit the route without the param.